compiler/src/org.graalvm.compiler.lir.aarch64/src/org/graalvm/compiler/lir/aarch64/AArch64Move.java

```diff
@@ -517,6 +518,9 @@ private static void reg2reg(@SuppressWarnings("unused") CompilationResultBuilder
         final int size = Math.max(kind.getSizeInBytes() * Byte.SIZE, 32);
         if (kind.isInteger()) {
             masm.mov(size, dst, src);
+        } else if (size == 128) {
```

Comment:
What about moving a 64bit value into a vector register?

Reply:
That should be handled correctly by `masm.fmov`